### PR TITLE
Fix #1435: ContentBlockerTests failing intermittently.

### LINKED ISSUE #1435: ContentBlockerTests failing intermittently
This only happens on Travis, in like 10% of runs

### PR DESCRIPTION
**Pull request by iccub**
## Summary of Changes

This pull request fixes issue #1435 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/internal/issues/new?assignees=&labels=security%2C+security-reviews&template=security-review.md&title=iOS:%20Security+Review+for) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Comment by kylehickinson** [comment]
Should probably force travis to rebuild this PR a few times before merging :)

**Comment by iccub** [comment]
lol good idea :D will do, adding blocked label

**Comment by iccub** [comment]
@kylehickinson I ran CI 5 times and always passed fine, waiting for your approval :)

**Review by kylehickinson — APPROVED** [review]

**Pull request merged**
